frontend/app/package.json

```diff
@@ -3,25 +3,20 @@
   "version": "1.39.0",
   "license": "Apache-2.0",
   "private": true,
+  "type": "module",
   "homepage": "./",
   "scripts": {
     "analyze": "source-map-explorer 'build/static/js/*.js'",
-    "start": "env NODE_OPTIONS=--max_old_space_size=8192 ESLINT_NO_DEV_ERRORS=true craco start",
-    "build": "env NODE_OPTIONS=--max_old_space_size=8192 GENERATE_SOURCEMAP=false craco build",
-    "buildFast": "env NODE_OPTIONS=--max_old_space_size=8192 GENERATE_SOURCEMAP=false BUILD_AS_FAST_AS_POSSIBLE=true craco build",
-    "test": "env NODE_OPTIONS=\"--experimental-worker --max_old_space_size=8192\" craco test --watchAll=false --env=../jest/jsdom-polyfill-env.js --runTestsByPath ${TESTPATH}",
-    "testWatch": "env NODE_OPTIONS=\"--experimental-worker --max_old_space_size=8192\" craco test --env=../jest/jsdom-polyfill-env.js --runTestsByPath ${TESTPATH}",
+    "start": "vite",
+    "build": "tsc && vite build",
```

Comment:
Do you know if it is actually required to run `tsc` here? The [vite docs](https://vite.dev/guide/features#transpile-only) only mention that running `tsc --noEmit` might be beneficial to check for typing issues before production build.

Reply:
+1 seems like it shouldn't be needed and would hamper our perf gains to run this for all builds

Reply:
and if we do want typechecking, i think we should do the `--noEmit` as lukas suggested so that we're not doing extra work twice

Reply:
Will remove because we have CI that runs the typecheck that would block release.